src/index.html

```diff
@@ -1,13 +1,312 @@
+          around, and feel their vibe
+        </p>
+      </div>
+      <div class="interested__card">
+        <img
+          src="images/vc3.png"
+          alt="fitness logo"
+          class="interested__img"
+        >
+        <p class="interested__name">fitness</p>
+        <p class="interested__text">
+          Combine business with <br>
+          pleasure, and discover countless<br>
+          ways to stay fit while playing<br>
+          your favorite VR Games!
+        </p>
+      </div>
+      <div class="interested__card">
+        <img
+          src="images/vc4.png"
+          alt="social interactions logo"
+          class="interested__img"
+        >
+        <p class="interested__name">social interactions</p>
+        <p class="interested__text">
+          Hang out with your friends in<br>
+          the virtual world when you can’t<br>
+          meet space requirements
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="about">
+    <div class="about__title">
+      <div class="about__box">
+        <p class="about__name">About <span class="about__name--accent">product</span></p>
+        <p class="about__text">KAT loco is a foot-based VR locomotion system that gives complete physical<br>
+          control over lower-body actions, allowing you to freely walk, run, and carry out<br>
+          just any other movement in virtual reality.</p>
+      </div>
+      <button class="button__play">Play Video</button>
+    </div>
+    <div class="about__vr">
+      <div class="vr__left">
+        <p class="left__par">Uniwersally<br>compatible</p>
+        <p class="left__par">User friendly<br> hardware</p>
+        <p class="left__par">Complete SDK <br>for Developers</p>
+      </div>
+      <div class="vr__center">
+        <img
+          src="images/aboutt.png"
+          alt="photo of vr"
+          class="vr__img"
+        >
+      </div>
+      <div class="vr__right">
+        <p class="right__par">Complete<br> locomotion system</p>
+        <p class="right__par">Smart wireless<br> sensor</p>
+        <p class="right__par">Multifunctional<br> software</p>
+      </div>
+    </div>
+    <div class="about__kat">
+      <img
+        src="images/nice.png"
+        alt="person who game VR"
+        class="nice__img"
+      >
+      <div class="about__nice">
+        <p class="nice__hello">Hello,</p>
+        <p class="nice__title">Nice to meet <span class="nice__title--accent">you!</span></p>
+        <p class="nice__text">KAT VR is an independent company dedicated to the research,<br>
+          development, and sales of VR Locomotion products and solutions.<br>
+          Founded in 2013, we have quickly grown to become one of the<br>
+          world’s leading professional suppliers of VR games’ & simulations’<br>
+          equipment</p>
+      </div>
+    </div>
+  </div>
+  <div class="tech">
+    <p class="tech__label"> Tech <span class="tech__label--accent"> specs </span>
+    </p>
```

Comment:
```suggestion
    <p class="tech__label">
        Tech <span class="tech__label--accent"> specs </span>
    </p>
```
Гадаю, так буде трохи ліпше